dali/test/python/test_dali_tf_dataset_mnist_eager.py

```diff
@@ -29,7 +29,7 @@ def test_keras_single_cpu():
     run_keras_single_device('cpu', 0)
 
 
-@raises(Exception)
+@raises(Exception, regex="TF device and DALI device mismatch")
```

Comment:
Why not use glob?

Reply:
Because with regex it automatically matches it anywhere - I don't have to put `*`.

Reply:
In case of glob you can omit leading and trailing as well. In that case it should be equivalent.

Reply:
Yes, globs behave the same way.

Reply:
So, they are not really globs, are they?

Reply:
https://docs.python.org/3/library/fnmatch.html#fnmatch.translate is used to translate glob into regexp. And then it is used to find any match, not necessarily whole string.